modules/nw-multus-host-device-object.adoc

```diff
@@ -39,14 +56,31 @@ parameters: `device`, `hwaddr`, `kernelpath`, or `pciBusID`.
   "pciBusID": "<pciBusID>" <5>
 }
 ----
-<1> `name`: Specify the name of the `NetworkAttachmentDefinition`.
+<1> Specify the value for the `name` parameter you provided previously for
+the CNO configuration.
 
-<2> `device`: Specify the name of the device, such as `eth0`.
+<2> Specify the name of the device, such as `eth0`.
 
-<3> `hwaddr`: Specify the device hardware MAC address.
+<3> Specify the device hardware MAC address.
 
-<4> `kernelpath`: Specify the Linux kernel device path, such as
+<4> Specify the Linux kernel device path, such as
 `/sys/devices/pci0000:00/0000:00:1f.6`.
 
-<5> `pciBusID`: Specify the PCI address of the network device, such as
-`0000:00:1f.6`.
+<5> Specify the PCI address of the network device, such as `0000:00:1f.6`.
+
+[id="nw-multus-hostdev-config-example_{context}"]
+== host-device configuration example
```

Comment:
Initial cap? This looks odd.

Reply:
@mburke5678, it does feel wrong. I asked in #forum-multus and all the Multus CNI plug-ins are lowercase names. I'm not completely surprised, as it is a plug-in rather than a complex software project or product.